book/src/advanced/prover.md

```diff
@@ -4,10 +4,19 @@ vlayer `Prover` contracts are almost the same as regular Solidity smart contract
 
 - **Access to Off-Chain Data:** `Prover` contracts accept data from multiple sources through features such as [time travel](/features/time-travel.html), [teleport](/features/teleport.html), [email proofs](/features/email.html), and [web proofs](/features/web.html). This allows claims to be verified on-chain without exposing all input the data.
 
+<!-- FEEDBACK: one could think that Verifier also works on zkEVM, this "where the proofs of computation ..." were a bit confusing for me. -->
 - **Execution Environment:** The `Prover` code executes on the vlayer zkEVM, where the proofs of computation are subsequently verified by the on-chain `Verifier` contract. Unlike the on-chain contract, the `Prover` does not have access to the current block. It can only access previously mined blocks. Under the hood, vlayer generates zero-knowledge proofs of the `Prover`'s execution.
 
 ## Prover in-depth
 
+<!-- FEEDBACK: Few questions that I was looking answers in more details here:
```

Comment:
Proposed update in separate PR https://github.com/vlayer-xyz/vlayer/pull/2462
@akoszowski